src/Runner/CliTester.php

```diff
@@ -228,9 +230,6 @@ private function createRunner()
 	/** @return string */
 	private function prepareCodeCoverage()
 	{
-		if (!$this->interpreter->hasXdebug()) {
-			throw new \Exception("Code coverage functionality requires Xdebug extension (used {$this->interpreter->getCommandLine()})");
-		}
```

Comment:
That doesn't make any sense. PHPDBG may or may not XDebug loaded, just like normal CLI/CGI SAPI.